### DISCUSSION TITLE
OpenZeppelin 的在线合约生成器

### DISCUSSION THREAD

**Discussion by liushooter** [rebase-network/hello-world#73] [Show and tell]
> ![](https://user-images.githubusercontent.com/1235106/158725477-7cb71ac6-d6db-4d04-9933-1e41f41cfa7c.png)
https://wizard.openzeppelin.com/

通过这个 web app 可以从 OpenZeppelin 的合约组件中进行选择，比如 ERC20 合约可以添加 Mintable 和 Votes，甚至还能在 token 层面添加 flash loan（Flash Minting 选项）

**Comment by wartstone** [comment]
确实合约开发相比web2开发更范式，web2目前在推的低代码, web3貌似能更快，如果这样solidity开发者们会不会虚~
